server/database-postgres/helpers.js

```diff
@@ -0,0 +1,97 @@
+const axios = require('axios');
+const { db, User, Language, UserLanguage } = require('./models.js');
+
+const getUserById = async (userId) => {
+  try {
+    const user = await User.findOne({ where: { userId } });
+    // console.log('😂 getUserById function result', user.dataValues);
+    return user;
+  } catch (err) {
+    console.error(err);
+    return null;
+  }
+};
+
+const getUserLanguagesByUserId = async (userId) => {
+  try {
+    const languages = await UserLanguage.findAll({ where: { userUserId: userId }})
+      .then(ids => ids.map(id => id.languageLanguageId))
+      .then(languageIds => Language.findAll({ where: {languageId: languageIds}}))
+      .then(languageNames => languageNames.map(lang => lang.name))
+      .catch(error => console.log(error));
+
+      return languages;
+  } catch (err) {
+    console.log(err);
+    return null;
+  }
+}
+
+const getUserNameAndPhoto = async (userId) => {
+  try {
+    const { name, avatarUrl } = await User.findOne({ where: { userId } }, 'name avatarUrl');
+    // console.log('🎷 getUserNameAndPhoto result', name, avatarUrl)
+    return { name, avatarUrl };
+  } catch (err) {
+    return null;
+  }
+};
+
+const getUserSuperhostStatus = async (userId) => {
+  try {
+    const { isSuperhost } = await User.findOne({ where: { userId } }, 'isSuperhost');
+    // console.log('🥁 getUserSuperhostStatus function result', isSuperhost)
+    return { isSuperhost };
```

Comment:
Is there a reason you are returning an object here?

Reply:
front end required it to be an object.